pandas/io/formats/style_render.py

```diff
@@ -124,6 +128,7 @@ def __init__(
         self.hide_columns_: list = [False] * self.columns.nlevels
         self.hidden_rows: Sequence[int] = []  # sequence for specific hidden rows/cols
         self.hidden_columns: Sequence[int] = []
+        self.descriptors: list[str | Callable | tuple[str, Callable]] = []
```

Comment:
If you do stick with the tuple, it should be `tuple(str, str | Callable)` which would allow someone to pass a custom name for the output along with a common function, e.g., `("average", "mean").  If moving to a dict, then both `{"average":"mean"}` and `{"average":Series.mean}` should be allowed.

Reply:
yes, agreed, having a think if this can be improved.